lib/src/qr_scanner_overlay_shape.dart

```diff
@@ -3,14 +3,15 @@ import 'dart:ui';
 import 'package:flutter/material.dart';
 
 class QrScannerOverlayShape extends ShapeBorder {
-  QrScannerOverlayShape({
-    this.borderColor = Colors.red,
-    this.borderWidth = 3.0,
-    this.overlayColor = const Color.fromRGBO(0, 0, 0, 80),
-    this.borderRadius = 0,
-    this.borderLength = 40,
-    this.cutOutSize = 250,
-  }) : assert(
+  QrScannerOverlayShape(
+      {this.borderColor = Colors.red,
+      this.borderWidth = 3.0,
+      this.overlayColor = const Color.fromRGBO(0, 0, 0, 80),
+      this.borderRadius = 0,
+      this.borderLength = 40,
+      this.cutOutSize = 250,
+      this.cutOutBottomOffset = 0})
```

Comment:
```suggestion
      this.cutOutBottomOffset = 0,})
```
and `dartfmt` please.